tests/ssl-opt.sh

```diff
@@ -1474,18 +1487,18 @@ run_test    "Max fragment length, DTLS: enabled, default, larger message" \
 requires_config_disabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
 run_test    "Max fragment length: disabled, larger message" \
             "$P_SRV debug_level=3" \
-            "$P_CLI debug_level=3 request_size=16385" \
+            "$P_CLI debug_level=3 request_size=$(expr $MAX_CONTENT_LEN + 1)" \
             0 \
             -C "Maximum fragment length is 16384" \
             -S "Maximum fragment length is 16384" \
```

Comment:
Shouldn't it be:
`-C "Maximum fragment length is $MAX_CONTENT_LEN" \`
`-S "Maximum fragment length is $MAX_CONTENT_LEN" \`
?

Reply:
No, because in this test (and the one mentioned below) MBEDTLS_SSL_MAX_FRAGMENT_LENGTH is disabled on the mbedTLS side (see a few lines above), so mbedTLS can't negotiate a smaller fragment length than the default (even if the other end requests it).

I'll add a comment to this effect, as it's not immediately obvious what's happening here.